lldb/unittests/SymbolFile/DWARF/Inputs/DW_AT_spec_decl_exists-test.yaml

```diff
@@ -0,0 +1,675 @@
+# struct Type {};
+# 
+# template <typename _Tp, bool, bool, bool> struct _Optional_payload;
+# 
+# template <typename _Tp> struct _Optional_payload<_Tp, true, false, false> {};
+# 
+# template <typename _Tp, bool _Copy, bool _Move>
+# struct _Optional_payload<_Tp, false, _Copy, _Move>
+#     : _Optional_payload<_Tp, true, false, false> {};
+# 
+# int main() {
+#   _Optional_payload<Type, false, false, true> X;
+# }
+#
+# YAML generated on Linux using obj2yaml on the above program
+# compiled with G++.
```

Comment:
Lets add a comment saying this is malformed DWARF that is missing `DW_TAG_template_value_parameter` entries, which is important for the test because that makes the two specializations structurally match in the AST.